JS/app.js

```diff
@@ -0,0 +1,214 @@
+console.log("hello");
+
+//-----------------Easy Going-------------//
+//write a loop that will log the numbers 1 through 20
+
+for (let i = 1; i<=20; i++){
+	console.log(i);
+}
+
+//---------------Get even-------------//
+// //write a FOR loop that will only log the even numbers in 0 through 200
+
+for (let i = 2; i<=200; i+=2) {
+	if(i % 2 === 0){	
+	console.log(i);
+}
+}
+//----------------Excited Kitten----------------//
+//1.Write code that logs "Loves me, pet me! HSSSSSSS!" 20 times
+//2.For every even number in your loop, log "...human...why you taking pictures of me?...",
+// "...the catnip made me do it...", or "...why does the red dot always get away..." at random.
+
+
+
+//created variables that hold the strings
+const catOne = "...human...why you taking pictures of me?...";
+const catTwo = "...the catnip made me do it...";
+const catThree = "...why does the red dot always get away...";
+
+
+//loop that logs "Love me, pet me! HSSSSSS!" 20 times
+for (let i = 0; i <= 20; i++){
+			console.log("Love me, pet me! HSSSSSS!");
+//check to see if the number that i is going through
+	if(i % 2 === 0){
+		//made variable that holds the function Math.random
+		let randCat = Math.random();
+		//if the random number generated is less than equal to .33, log catOne
+	  if (randCat <= .33){
+	  	console.log(catOne);
+	  			//if the random number generated is less than equal to .66, log catTwo
+	} else if (randCat <= .66) {
+		console.log(catTwo);
+		//any other number above .66, log catThree
+	} else {
+		console.log(catThree);
+	}
+	}
+}
+
+
+
+// Write a javascript application that logs all numbers from 1 - 100.
+// If a number is divisible by 3 log "Fizz" instead of the number.
+// If a number is divisible by 5 log "Buzz" instead of the number.
+// If a number is divisible by 3 and 5 log "FizzBuzz" instead of the number.
+
+// setting up loop
+
+for (let i = 1; i < 100; i++) {
+	//if i is divisible by 3 and 5, console log "fizzbuzz" instead of the number
+	if (i % 3 && 5 === 0)
```

Comment:
this isn't going to work --- it's not valid syntax.  try `if (i % 3 === 0 && i % 5 === 0)` because that means "if i is evenly divisible by 3 AND ALSO i is evenly divisible by 5.

good job with the problem otherwise!

Reply:
also, check your output of problems to make sure they're working like you expect